crypto/s2n_ecc_evp.c

```diff
@@ -478,21 +480,25 @@ int s2n_ecc_evp_parse_params_point(struct s2n_blob *point_blob, struct s2n_ecc_e
     return 0;
 }
 
-int s2n_ecc_evp_parse_params(struct s2n_ecdhe_raw_server_params *raw_server_ecc_params,
-                             struct s2n_ecc_evp_params *ecc_evp_params) {
+int s2n_ecc_evp_parse_params(struct s2n_connection* conn,
+                                struct s2n_ecdhe_raw_server_params* raw_server_ecc_params,
+                                struct s2n_ecc_evp_params* ecc_evp_params) {
     S2N_ERROR_IF(
-        s2n_ecc_evp_find_supported_curve(&raw_server_ecc_params->curve_blob, &ecc_evp_params->negotiated_curve) != 0,
-        S2N_ERR_ECDHE_UNSUPPORTED_CURVE);
+            s2n_ecc_evp_find_supported_curve(conn, &raw_server_ecc_params->curve_blob, &ecc_evp_params->negotiated_curve) != 0,
+            S2N_ERR_ECDHE_UNSUPPORTED_CURVE);
```

Comment:
Nit: If you're going to touch this code, can you switch this to POSIX_ENSURE instead of S2N_ERROR_IF? You just need to invert the condition.